src/getMachineId.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+from decEncoder import *
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Dynamo resource"""
+dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
```

Comment:
Something I realized the other day is that Lambda lets you get the AWS region through environment variables by default. Check this out: http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html. Maybe in a future update we can update this so it's not hardcoded.